docker-compose.yml

```diff
@@ -20,8 +20,10 @@ services:
 
   postgres:
     image: postgres:12.2-alpine
+    networks:
+      [webservice_network]
     volumes:
-      - ./pgdata/:/var/lib/postgresql/data/devroast
+      - devroast:/var/lib/postgresql/data/
```

Comment:
Why is this updating to have a named volume with no definition instead of a directory?

Reply:
I found that using a Docker volume instead of hard directory solved earlier problems with bootstrapping the container.